eternum-docs/src/components/QuestRewards.tsx

```
@@ -0,0 +1,32 @@
+import ResourceIcon from "./ResourceIcon";
+
+function addSpacesBeforeCapitals(str: string): string {
+  return str.replace(/([A-Z])/g, " $1").trim();
+}
+
+export default function QuestRewards() {
+  return (
+    <>
+      {Object.entries(EternumGlobalConfig.questResources).map(([questType, rewards]) => {
+        return (
+          <div className="my-4 p-3 ">
```

Biome (1.9.4)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)